source/microbit/microbitconstimagetuples.c

```diff
@@ -0,0 +1,58 @@
+/*
+ * The MIT License (MIT)
```

Comment:
Ownership conflicts with my trotskyist beliefs :wink: 
Seriously, could we give the copyright to the PSF, and not have to worry about the legal crap?